### PR TITLE
Auth Frontend: remove github login page

### LINKED ISSUE #1070: Auth Frontend: Remove Github login page
Remove the temporary GitHub login page 

https://github.com/spotify/backstage/issues/1002

### PR DESCRIPTION
**Pull request by soapraj**
removing the github login page in favor of the github auth flow added [here](https://github.com/spotify/backstage/pull/1074) 

#### :heavy_check_mark: Checklist

- [ ] All tests are passing `yarn test`
- [ ] Screenshots attached (for UI changes)
- [ ] Relevant documentation updated
- [ ] Prettier run on changed files
- [ ] Tests added for new functionality
- [ ] Regression tests added for bug fixes

### CODE REVIEW

File: plugins/catalog/package.json [hunk 1]
```diff
@@ -19,6 +19,7 @@
   "dependencies": {
     "@backstage/core": "^0.1.1-alpha.6",
     "@backstage/theme": "^0.1.1-alpha.6",
+    "@backstage/catalog-model": "^0.1.1-alpha.6",
```
Comment (stefanalund) [inline plugins/catalog/package.json:22]: Why this change?

**Review by stefanalund — APPROVED** [review]

File: plugins/catalog/package.json (hunk 1 above)
Comment (soapraj) [reply plugins/catalog/package.json:22]: The catalog plugin refers to a type from the catalog-model [here](https://github.com/spotify/backstage/blob/master/plugins/catalog/src/data/utils.ts#L17) and the dep is not present  in [package.json](https://github.com/spotify/backstage/blob/master/plugins/catalog/package.json) in master 

I ran `yarn run lint` locally on my branch and master and it errored out saying the dependency is not added to package.json. 

Something is up with the linting setup in GitHub workflows maybe?

Maybe @freben @Rugvip can take a look at it.

Comment (Rugvip) [reply plugins/catalog/package.json:22]: Seems like a correct fix, but make a separate PR for it imo.

**Review by stefanalund — APPROVED** [review]
